package-lock.json

```diff
@@ -5,6 +5,7 @@
   "requires": true,
   "packages": {
     "": {
+      "name": "aws-lambda-powertools-typescript",
```

Comment:
Could you please revert this? Since the change is in the unit tests it shouldn't affect the lock file.

Reply:
Hello @dreamorosi  yes, for sure. This was automatically added when I execute npm install , I'm using the npm version 7.24.2.  
Previously I was using the version 6 of npm and it broke completely the npm lock as you can see in the attached screenshot. It would be good to add the check of having at least npm v7  in the contributors section. 
<img width="1248" alt="Screenshot 2022-01-10 at 6 16 41 PM" src="https://user-images.githubusercontent.com/18292766/149195353-ffd138b8-d134-4a5b-a0cd-c608455d1a2a.png">

Reply:
Thanks, yes we are working on standardising the guidelines about which node/npm versions are recommended so that this can be avoided. In the meantime thanks for reverting it.